tests/test_functions.py

```diff
@@ -92,6 +92,11 @@ def test_remove_nth():
     assert func.remove_nth(l, 2) == [3, 5, 67, -4]
     assert func.remove_nth(l) == func.remove_nth(l, 2)
     assert func.remove_nth(l, 1) == []
+    # the next assertion fails as the list, 
+    # previously altered by the other operations is already empty
+    # we can either perform the check with the original list or change 
+    # the expected value
```

Comment:
A third option would be to write the function so that it does not alter the list :smiley: